scripts/generate-acknowledgements.sh

```diff
@@ -0,0 +1,57 @@
+#!/bin/bash
+set -euo pipefail
+
+if [[ ! -f "./go.mod" ]]; then
+    echo "Need to run this from the directory containing go.mod"
+    exit 1
+fi
+
+OUTPUT_FILE="ACKNOWLEDGEMENTS.md"
+
+# Ensure modules are downloaded
+go mod download
+
+# Get go-licenses tool
+go install github.com/google/go-licenses@latest
+GO_LICENSES="${HOME}/go/bin/go-licenses"
```

Comment:
Some people might have installed this by some other means. Some, not necessarily the same, people might also have a different GOPATH.

```suggestion
if ! command -v go-licenses >/dev/null; then
	go install github.com/google/go-licenses@latest
	GO_LICENSES="$(go env GOPATH)/bin/go-licenses"
else
	GO_LICENSES="$(command -v go-licences)"
fi
```

Reply:
This is all the testing I've done:
```
if ! command -v go-licenses >/dev/null; then    go install github.com/google/go-licenses@latest;   GO_LICENSES="$(go env GOPATH)/bin/go-licenses"; else    GO_LICENSES="$(command -v go-licenses)"; fi
[narthana@quirinius ~]$ echo $GO_LICENSES
```